tasks/managed/filter-already-released-advisory-images/filter-already-released-advisory-images.yaml

```diff
@@ -148,36 +148,39 @@ spec:
 
         echo "$SNAPSHOT_JSON" | jq -c '.components[]' | while IFS= read -r component; do
             CONTAINER_IMAGE=$(echo "$component" | jq -r '.containerImage')
-            RH_REGISTRY_REPO=$(echo "$component" | jq -r '.["rh-registry-repo"] // .repository')
             COMPONENT_NAME=$(echo "$component" | jq -r '.name')
 
             echo "Processing component: $COMPONENT_NAME with image: $CONTAINER_IMAGE" >&2
 
-            # Use get-image-architectures utility (same approach as populate-release-notes)
-            if get-image-architectures "${CONTAINER_IMAGE}" | while IFS= read -r arch_json; do
-                if [[ -n "$arch_json" ]]; then
-                    # Extract architecture-specific digest from arch_json
-                    arch_digest=$(echo "$arch_json" | jq -r '.digest')
-                    arch=$(echo "$arch_json" | jq -r '.platform.architecture')
-
-                    # Create transformed component with registry format containerImage
-                    TRANSFORMED_CONTAINER_IMAGE="${RH_REGISTRY_REPO}@${arch_digest}"
-                    echo "$component" \
-                      | jq \
-                        --arg ci "$TRANSFORMED_CONTAINER_IMAGE" \
-                        --arg repo "$RH_REGISTRY_REPO" \
-                        --arg orig_name "$COMPONENT_NAME" \
-                        --arg arch "$arch" \
-                        '.containerImage = $ci
-                         | .repository = $repo
-                         | .originalComponent = $orig_name
-                         | .architecture = $arch'
-                fi
-            done; then
-                echo "Successfully transformed component: $COMPONENT_NAME" >&2
-            else
+            if ! digests=$(get-image-architectures "${CONTAINER_IMAGE}"); then
                 echo "Warning: Failed to resolve architectures for ${CONTAINER_IMAGE}, skipping component" >&2
+                continue
             fi
+
+            repositories=$(echo "$component" | jq -c '.repositories')
```

Comment:
typically shellcheck tells you to do `jq -c '.repositories' <<< "$component"` instead of echoing it. I am not sure why it doesn't complain here (same in other spots)

Reply:
Fixed.